.github/workflows/clair.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: [ubuntu-latest]
     steps:
     - uses: actions/checkout@v3
```

Comment:
not related to this PR, but this whole workflow should probably be removed entirely as a follow-up
we moved to Trivy in antrea-io/antrea